doc/meetings/build-wg-meeting-2019-11-19.md

```diff
@@ -0,0 +1,77 @@
+﻿# Node.js Foundation Build WorkGroup Meeting 2019-11-19
+
+## Links
+
+
+* **Recording**:  https://www.youtube.com/watch?v=pm3fuQM5Yow&feature=youtu.be
+* **GitHub Issue**: https://github.com/nodejs/build/issues/2046
+
+## Present
+
+
+* Michael Dawson
+* Richard Lau
+* Sam Roberts
+* Ashley Cripps
+* Rod Vagg
+* Dave Ings
+
+## Agenda
+
+## Announcements
+ 
+*Extracted from **build-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+
+* Experiment with Docker
+  * Allow collabs to more easily manage setup, shrink the set of virtual machines that we 
+    need to maintain
+  * Problem was with parallel compiles, we get occasional failures. Something about make
+    in docker that hits race condition.  
+  * Michael different dockers hosts? Rod, all ubuntu 18.04 so possibly related to that.
+  * New version of docker might help.
+  * Some bugs on make issue tracker related to parallelism, but different versions used.
+  * Occurs across different containers versions
+  * To recreate, start docker, clone, run make build with -j4.  Often in about 1/20. But could
+    related to running under jenkins
+
+
+* Compressed pointers
+  * Google switching chrome over to use compressed pointers.
+  * May re-start discussion of multiple versions of binaries
+    * significant memory benefits
+    * build time flag, so a potential concern to make compressed version only option
+  * Google has not said they are removing option, but there is concern we’ll get 
+    a lot less test coverage as only the compressed option will be used in Chrome were
+    the gc gets a lot of its real-world test coverage from. 

```

Comment:
* PR to add build option to Node.js to enable compressed pointers: https://github.com/nodejs/node/pull/30463

Reply:
* Background doc on compressed pointers in V8 https://docs.google.com/document/d/10qh2-b4C5OtSg-xLwyZpEI5ZihVBPtn1xwKBbQC26yI
_(I dropped this in the chat, as Richard did the PR link, so these were part of the meeting)_